portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,31 @@
   </head>
   <body>
     <div id="content">
-      <img src="sheon_images/headshot.jpg" />
+      <img src="sheon_images/headshot.jpg" width="500" height="600"/>
       <h1>Sheon Mwapinza <a href="https://www.linkedin.com/in/sheon-mwapinza-963518196/">LinkedIn</a>, <a href="https://github.com/sheonkuda">GitHub</a>.</h1>
-      <p>My name is Sheon Mwapinza, I am from Zimbabwe! I am a sophomore at the University of Pennsylvanina 
-          and I am studying electrical engineering. I love sports, playing chess and I love playing with my baby brother.</p>
-      <p>My journey in computer science has been most fulfulling. Nothing compares to the feeling that mauls you after spending an entire night
-         awake working on the SPS portfolio and are finally done. Although painful, there is a silver lining to spending these nights awake,
+      <p>Quote of the day(see bottom of page):</p>
+      <button onclick="addRandomGreeting()">Quote</button>
+      
+      <p> 
+          Hailing from the "City of Progress" Gweru, Zimbabwe! Growing up I dreamt of the Gweru that could be but,
+          it did not take me long to realise that everyone had the same dream and noone seemed to be doing anything to take us there, and that's when I decided to take matters into my own hands.
+          For the longest time I longed to transform my community through engineering which has led me to pursue an electrical engineering at the University of Pennsylvania. When I am not thinking about 
+          how I will change Zimbabwe, I play sports, chess, and I love playing with my baby brother.
+      </p>
+      <p>
+          Starting last year I embarked on a journey to provide a platform for the underappreciated arts in Zimbabwe 
+          so to raise awareness and motivate people to venture into the these disciplines. Together with three of my friends we started Zimries- 
+          Zimbabwean Diaries, a platform where people can share their poetry, articles, speaches and bring the Zimbabwean voice to the world. Inspired by 
+          by the work people were doing and the change and influence they were having in society, I wanted to also bring myvoice to the world and make an impact.
+          I wrote my first article, <a href="https://zimries.wordpress.com/2020/09/04/the-africa-we-want/">My African dream</a>. This marked the beginning of my journey as a writer, but I soon
+          realised I did not like writing and was also the end. 
+      </p>
+      <p>
+         My journey as a programmer has been most fulfulling. Nothing compares to the feeling that mauls you after spending an entire night
+         awake working on the SPS portfolio and are finally done. Although painful, there is a silver lining to these sleepless nights,
          every hour spent working on the projects, homeworks, and assignemnt means more knowledge and experience gained. Some of the projects I 
-         have worked on are:</p>
+         have worked on are:
```

Comment:
Are there links to these projects? They sound really cool, and I would love to see them

Reply:
unfortunately no